### PR TITLE
Adding links to ktlint demo to the README.md

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
Continuation from https://github.com/pinterest/ktlint/issues/912. Added a section with ktlint online demo. This can attract more users to ktlint. As now they can see how powerful ktlint is without installing it to your PC.

### CODE REVIEW

**Comment by akuleshov7** [comment]
@Tapchicoma is it fine?

File: README.md [hunk 1]
```diff
@@ -117,6 +117,12 @@ disabled_rules=import-ordering
 disabled_rules=indent
 ```
 
+## Online demo
+In case you would like to see how `ktlint` works and you are not able (or do not want) to install it -
```
Comment (shashachu) [inline README.md:121]: I'm fine with the link. Can you please change the language to:

```
## Online demo
You can try `ktlint` online [here](https://ktlint-demo.herokuapp.com/) using the standard or a custom ruleset. To contribute, please visit the [GitHub repository](https://github.com/akuleshov7/diKTat-demo).
```

Comment (akuleshov7) [reply README.md]: updated the text

Comment (Tapchicoma) [reply README.md]: LGTM

**Review by Tapchicoma — APPROVED** [review]

**Pull request merged**